### PR TITLE
Contact me form

### PR DESCRIPTION
**Pull request by Sonikak004**
### **In the following Branch** :

I have created contact me form using _html_ and _css_ and used formspree to generate and record forms.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -241,6 +240,61 @@ <h3 class="about-head-text">Github</h3>
           <h3 class="about-head-text">Codepen</h3>
           <h3 class="about-head-text">Gitlab</h3>
           <h3 class="about-head-text">Terminal</h3>
+        </div>
+
+          <!-- contact me form -->
+
+          <footer>
+          <div class="contact-me">
+            <h1 class="contact-me-para">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
```
Comment (bimbolabuari) [inline index.html]: [Optional] - For semantic purposes and search engine optimization, kindly use just one h1 in an HTML file, other headings can be h2-h6, depending on their order of importance.

**Review by bimbolabuari — COMMENTED** [review]
### This is a manual review

- Highlights:
  - All input fields are wrapped in a single form element.
  - All input fields have the correct `type` attribute
  - HTML validation is implemented.
  
# Request Changes 
- As required, button interactions such as hover, active (when it's pressed), disabled and enable (default state) should be implemented.
 - When I try submitting the contact values using Formspree I get a 404 error.


https://user-images.githubusercontent.com/66112760/208756371-35f2dcbf-c1ff-4d9c-a12c-e40e8f50cc9b.mov




Cheers and Happy coding!:clap::clap::clap:

**Comment by bellom** [comment]
@Sonikak004 - We are waiting on your changes.

**Comment by Sonikak004** [comment]
> @Sonikak004 - We are waiting on your changes.

hi, I will fix them quickly give me some time :)

**Review by bimbolabuari — COMMENTED** [review]
## This is a manual review

Highlights:
- Implemented the form submission using the Formspree service 🎉 

### Request Changes 
As previously requested, button interactions such as hover, active (when it's pressed), disabled and enable (default state) should be implemented. 🙂 

- Please check the images below for the button interaction colours and styles.
Screenshot 2022-12-21 at 16 31 22


   - Currently, on hover, the website has a white background without a coloured border and on pressed there is no button interaction, kindly implement this 👍 
    
Keep rocking 🥇

**Review by bimbolabuari — APPROVED** [review]
### This is a manual review

Note: It will be approved on your dashboard as soon as possible.

# Approved  👍 
Cheers and Happy coding!:clap::clap::clap:

**Pull request merged**